### ISSUE TITLE
[White Label -Image-svg ]: Multiple landmarks are announcing for whole Bot response.

### ISSUE THREAD

**Issue by Kusum199** [microsoft/BotFramework-WebChat#5724]
### What is the PWD impact?

Screen reader users may be confused when multiple landmarks are announced for a single bot response. This creates uncertainty about structure and hierarchy, increasing cognitive load and making conversational navigation more difficult.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

ARIA roles

### How to reproduce the issue?

Pre-Requisite:
Turn on Narrator

Repro Steps
1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3.Navigate to white label experience chat page and type Image-svg word in type your message text box.
4. Response will appear.
5. Navigate to the whole response Bot.
6. Observe an issue that Multiple landmarks are announcing for whole Bot response.

### What do you expect?

The element should expose only one meaningful landmark to assistive technologies. The bot response container should not inherit or apply multiple landmark roles. narrator should announce as Main landmark. 

### What actually happened?

Multiple landmarks are announcing for whole Bot response.
Observation: Main landmark and complementary landmark.

### WCAG reference

https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

### Adaptive Card JSON

```json

```
